dbms/src/Storages/Transaction/RegionTable.cpp

```diff
@@ -367,44 +367,6 @@ RegionDataReadInfoList RegionTable::tryFlushRegion(const RegionPtrWithBlock & re
     return data_list_to_remove;
 }
 
-RegionID RegionTable::pickRegionToFlush()
```

Comment:
Can we also remove `dirty_regions` and `shouldFlush` after removing this method?

Reply:
We can remove shouldFlush function，but dirty_regions is still be used in function like tryFlushRegion